alacritty/src/config/mod.rs

```diff
@@ -125,6 +125,7 @@ pub fn load(options: &Options) -> UiConfig {
 
 /// Attempt to reload the configuration file.
 pub fn reload(config_path: &Path, options: &Options) -> Result<UiConfig> {
+    debug!("Reloading configuration file from: {:?}", config_path);
```

Comment:
```suggestion
    debug!("Reloading configuration file from: {:?}", config_path);
    
```

I'd add a newline, just to create some separation.